integration-tests/src/user/mod.rs

```diff
@@ -240,6 +244,41 @@ pub trait User {
             vec![Action::Stake(StakeAction { stake, public_key })],
         )
     }
+
+    #[cfg(feature = "protocol_feature_nep366_delegate_action")]
+    fn meta_tx(
+        &self,
+        signer_id: AccountId,
+        receiver_id: AccountId,
+        relayer_id: AccountId,
+        actions: Vec<Action>,
+    ) -> Result<FinalExecutionOutcomeView, ServerError> {
+        let inner_signer =
```

Comment:
nit: consider using test utils:https://github.com/near/nearcore/blob/master/docs/practices/testing/test_utils.md#signatures

Reply:
I had to create a new util for `InMemorySigner`, as the existing utils creates an `InMemoryValidatorSigner` which can't sign arbitrary stuff. But I think adding the utility makes sense, so I did.